Library/Homebrew/utils/bottles.rb

```diff
@@ -29,16 +29,16 @@ def native_regex
       end
 
       def receipt_path(bottle_file)
-        path = Utils.popen_read("tar", "-tf", bottle_file).lines.map(&:chomp).find do |line|
-          line =~ %r{.+/.+/INSTALL_RECEIPT.json}
-        end
+        command = tar_command "tf", bottle_file, "*/*/INSTALL_RECEIPT.json"
```

Comment:
`"*/*/INSTALL_RECEIPT.json"` requires the option `--wildcards` with GNU `tar` on Linux.